tcms/testplans/templates/testplans/get.html

```diff
@@ -162,7 +166,22 @@ <h2 class="card-pf-title">
         </div>
         <div class="list-group-item-container container-fluid hidden">
             <div class="row">
-                <div class="col-md-12">Some test case widgets</div>
+                <div class="col-md-9">
+                    <div>{% trans 'Text' %}:</div>
+                    <textarea class="js-test-case-expand-text"></textarea>
```

Comment:
that should be a div, see testcases/get.html

Reply:
The text itself is Markdown and needs to be rendered as HTML. In other templates we do this via template tag which also handles filtering unwanted HTML to prevent XSS.

This needs to somehow be handled on the BE to avoid code duplication but I'm not sure how ATM. What do you think about a `Markdown.render` API method which will render server side and return the HTML to the caller ?